examples/generate.sh

```diff
@@ -114,16 +114,20 @@ echo "Generated ${MACHINEDEPLOYMENT_GENERATED_FILE}"
 # Get Cert-manager provider components file
 curl -L -o "${COMPONENTS_CERT_MANAGER_GENERATED_FILE}" https://github.com/jetstack/cert-manager/releases/download/v0.13.0/cert-manager.yaml
 
+# Get enhanced envsubst version to evaluate expressions like ${VAR:=default}
+curl -L -o envsubst-go https://github.com/a8m/envsubst/releases/download/v1.2.0/envsubst-`uname -s`-`uname -m`
+chmod +x envsubst-go
```

Comment:
Should this be in a specific folder ? like a tmp, or the examples folder, rather than where this script is executed from ?

Reply:
Yes, it should be like that :)
By the way, do you think that replacing "vanilla" envsubst entirely with the "enhanced" version could be a thing there?
With the PR we will use two different envsubst when one of them just supports both use cases.